utils/clients/tenderly.ts

```diff
@@ -173,6 +179,24 @@ async function simulateNew(config: SimulationConfigNew): Promise<SimulationResul
       [`${proposalVotesKey}.abstainVotes`]: '0',
     }
 
+    targets.forEach((target, i) => {
+      const id = hashOperationOz(target, values[i], calldatas[i], HashZero, HashZero)
+      governorStateOverrides[`_timestamps[${id}]`] = '2' // must be > 1.
+    })
+  } else if (governorType === 'arb') {
+    const proposalCoreKey = `_proposals[${proposalId.toString()}]`
+    const proposalVotesKey = `_proposalVotes[${proposalId.toString()}]`
+    governorStateOverrides = {
+      // using dummy 1337 here because this need to be L1 block number but simBlock is L2 block number
+      [`${proposalCoreKey}.voteStart._deadline`]: '1337',
+      [`${proposalCoreKey}.voteEnd._deadline`]: '1337',
```

Comment:
can't you use an explicit `undefined` here?

Reply:
changing to 1, can't be undefined as we need it to be in the past so the proposal can execute